model/task/db.go

```diff
@@ -772,6 +772,25 @@ func schedulableHostTasksQuery() bson.M {
 	return q
 }
 
+// ScheduledContainerTasksQuery is a query that filters for scheduled tasks that are waiting for a pod
+// to be allocated to them, sorted by when they were scheduled.
+func ScheduledContainerTasksQuery() bson.M {
+	return bson.M{
+		ActivatedKey:         true,
+		StatusKey:            evergreen.TaskUndispatched,
+		ExecutionPlatformKey: ExecutionPlatformContainer,
+
+		// Filter out tasks disabled by negative priority
```

Comment:
I don't think we need this comment. It should be clear just from the code itself.